src/tests/JIT/Directed/Convert/out_of_range_fp_to_int_conversions.cpp

```diff
@@ -153,18 +154,8 @@ extern "C" DLLEXPORT  uint64_t ConvertDoubleToUInt64(double x, FPtoIntegerConver
                 return (uint64_t)ConvertDoubleToInt64(x - int64_max_plus_1, CONVERT_MANAGED_BACKWARD_COMPATIBLE_ARM32) + (0x8000000000000000);
             }
         }
-
-    case CONVERT_MANAGED_BACKWARD_COMPATIBLE_X86_X64:
```

Comment:
What is the purpose of these changes?

Reply:
I believe the previous `JIT_Dbl2Ulng` helper function would convert a `double` to `ulong` in a way that was compatible with MSVC's conversion, though it was not according to IEEE specification. Intel's new conversion instructions will follow IEEE (undefined negative values go to UINT64 max etc). 

These changes help to align everything (JIT_Dbl2Ulng and the test cases for JIT_Dbl2Ulng must match etc.)